components/service-operator/apis/access/v1beta1/principal_types_test.go

```diff
@@ -14,6 +17,7 @@ import (
 var _ = Describe("Principal", func() {
 
 	var principal v1beta1.Principal
+	var principalWithSvcAccTrust v1beta1.Principal
```

Comment:
this feels like it should be a split into seperate `Context` blocks rather than a mix of principal and principalWithSvcAccTrust

Reply:
yeah had that feeling when doing it, will fix